development/prow-installer/cmd/installer/main.go

```diff
@@ -134,23 +139,23 @@ func main() {
 			saname = fmt.Sprintf("%.30s", saname)
 			_, err = crmClient.AddSAtoRole(saname, serviceAccount.Roles, readConfig.Project, nil)
 			if err != nil {
-				log.Errorf("Failed assign sa %s to roles, got: %w", saname, err)
+				log.Errorf("Failed assign sa %s to roles, got: %v", saname, err)
```

Comment:
Did it complain about `%w`? That is most likely related to the choice of golang version. I think this feature is not present until go-1.10 or go-1.11

Reply:
It does not complain but instead of printing the entire struct it returns the value of an error.